### PR TITLE
Feat: Create admin dashboard index and links

### PR DESCRIPTION
**Pull request by Dominicod**
Did driver navigator with Lucas
*Created Admin Dashboard
*Created Admin Links
*Deleted credit_card_expiration_date model validation

### CODE REVIEW

File: spec/features/admin/index_spec.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    within("#admin_title") do
+      expect(page).to have_content "Admin Dashboard"
+    end
+  end
+
+  it "I see a link to the admin merchants index (/admin/merchants)" do
+    visit admin_index_path
+
+    within("#admin_links") do
```
Comment (rebeckahendricks) [inline spec/features/admin/index_spec.rb:15]: I like this

**Review by rebeckahendricks — APPROVED** [review]
Looks good - we'll have to add the paths to the links eventually, but this satisfies our user stories so far for now.

File: config/routes.rb [hunk 2]
```diff
@@ -5,4 +5,5 @@
   resources :invoices
   resources :invoice_items
   resources :customers
+  resources :admin, only: [:index]
```
Comment (thayes87) [inline config/routes.rb:8]: we will need to build this out eventually

**Review by thayes87 — APPROVED** [review]
Confirmed merge

**Pull request merged**